l2tdevtools/review_helpers/cli.py

```diff
@@ -28,7 +31,7 @@ def RunCommand(self, command):
       command (str): command to run.
 
     Returns:
-      tuple[int, bytes, bytes]: exit code, stdout and stderr data.
+      tuple[int, str, str]: exit code, stdout and stderr strings.
```

Comment:
stdout and stderr strings => strings containing output written to stdout and stderr